### DISCUSSION TITLE
如何将自定义组件在设计器中注册使用

### DISCUSSION THREAD

**Discussion by sorryljt** [alibaba/formily#1714] [Ideas]
如题，基于Formily/antd的自定义组件规则鞋类一个自定义组件，如何能让其在编辑器中可用？有无相关文档提供？

**Comment by Pioneer-Linzi** [comment]
const DesignableField = createDesignableField({
  registryName: 'DesignableField',
  components: {  },
})


在 components 里注册

**Comment by cyt68** [comment]
> 如题，基于Formily/antd的自定义组件规则鞋类一个自定义组件，如何能让其在编辑器中可用？有无相关文档提供？

"基于Formily/antd的自定义组件规则"，请问这个规则你是在哪看到的呢？
